src/Data/Map/Ordered/OMap.purs

```diff
@@ -152,3 +166,16 @@ alter f k om = case mOldValue, mNewValue of
   mOldValue = lookup k om
 
   mNewValue = f mOldValue
+
+moveTo :: forall k v. Ord k => Int -> k -> OMap k v -> Maybe (OMap k v)
+moveTo i k m = do
+  v <- lookup k m
+  let
+    m' = delete k m
+  insertAt i k v m'
+
+moveLeft :: forall k v. Ord k => k -> OMap k v -> Maybe (OMap k v)
```

Comment:
Probably not important... but should this be `moveBefore` and `moveAfter`, or `movePrior` and `moveNext` 🤔 .